### PR TITLE
Github Action CI

### PR DESCRIPTION
**Pull request by scowalt**
[Building successfully!](https://github.com/scowalt/pcsx2-Vanguard/runs/923393865)

### CODE REVIEW

**Comment by NarryG** [comment]
I'll look into whatever happened to the build and get back to you.

**Comment by NarryG** [comment]
Looking into the build now. I'm theorizing it might be an issue with the debug configuration

**Comment by NarryG** [comment]
I'm an idiot & refactored some code that never got pushed.

**Comment by NarryG** [comment]
I've pushed the fixed code to the 50X branch and created the new vanguard branch. If you want to make this PR for both, feel free. Might be good to do that across the board since we're going to have the 50X branch and the 51X branch soonish, and that'll definitely involve breaking changes.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    - name: Checkout current build target
+      uses: actions/checkout@v2
+      with:
+        path: 'pcsx2-Vanguard'
+        submodules: 'recursive'
+    - name: Checkout RTCV peer dependency
+      uses: actions/checkout@v2
+      with:
+        repository: 'ircluzar/RTCV'
+        ref: '505'
```
Comment (scowalt) [inline .github/workflows/build.yml:17]: This `'505'` may need to be variable if we use the new branching strategy.

**Comment by scowalt** [comment]
This is building now on `Debug:Win32`!

**Pull request merged**